apps/association_result_annotation/dxapp.json

```diff
@@ -1,120 +1,144 @@
 {
   "name": "association_result_annotation",
   "title": "Annotate GWAS, PheWAS Associations",
-  "summary": "association_result_annotation",
+  "summary": "This app increases the depth of annotation information in an input GWAS tsv file.",
   "dxapi": "1.0.0",
-  "version": "0.0.1",
-  "categories": [],
+  "version": "1.0.0",
+  "details": {
+    "whatsNew": "*1.0.0: updated to ubuntu 14.04, added patterns/help to input/output spec, added regionalOptions/assetDepends;",
+    "upstreamAuthor": "Ritchie, MD",
+    "citations": []
+  },
+  "categories": ["GWAS"],
   "inputSpec": [
     {
-      "name": "input_file",
+      "name": "associations",
       "label": "Input Assocaition File",
       "class": "file",
-      "optional": false
+      "optional": false,
+      "patterns": ["*.tsv"],
+      "help": "A data file containing information from a genome wide association catalog."
     },
     {
-      "name": "sql_file",
-      "label": "SQLite file",
+      "name": "sql_database",
+      "label": "SQLite databse file",
       "class": "file",
+      "patterns": ["*.db"],
       "hidden": true,
-      "default": {
-        "$dnanexus_link":
-        {
-          "project": "project-BYpFk1Q0pB0xzQY8ZxgJFv1V",
-          "id": "file-BxGpVYQ0pB0vZkV6g5JVgz9F"
-      }
-      }
+      "help": "A SQLite3 databse file.",
+      "suggestions": [
+        {   
+          "project": "project-FGqF5B00ByvPXG4G87yQy23G",
+          "id": "file-FGzv6FQ0Byv9p5ZV6ZkgVv3K",
+          "path": "/GWAS_Annotations",
+          "name": "Ritchie Lab Software_dx: AWS East"
```

Comment:
May need to work with Ritchie lab to decide what "name" should be used for suggestions